package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "client-oauth2",
-  "version": "4.3.2",
+  "version": "4.4.0",
```

Comment:
Why minor vs patch? I don't see any new features here...

Reply:
Because it has a few behavioural changes described in the PR description:
1. Preserving query string params;
2. Handling empty scopes.

Even though these are not features, I assumed that some existing projects may rely on the old and incorrect behaviour. Thus when when these projects silently update to a new patch version, they may brake.

I know my version doesn't strictly follow Semantic Versioning, but I think these changes should be made notable somehow.

Reply:
I see what you mean but 1. is clearly a bug as the second `?` would make the server not understand all the query params and thus fail. Then 2. is not as clear-cut but I would go [by the spec](https://tools.ietf.org/html/rfc6749#section-3.3) and since one can still send empty scopes by setting it to `""` we wouldn't be breaking any voluntary empty scope usage.

So my comment, I thing this deserves a PATCH release more than a MINOR.

Reply:
Ok, I changed it to patch.